openapi.yaml

```diff
@@ -126,25 +226,15 @@ components:
           scopes:
             full: Full access to the API
   parameters:
-    pageNumber:
-      name: page[number]
-      in: query
-      required: false
-      description: Page number of results
-      schema:
-        type: integer
-        minimum: 1
-        default: 1
-    pageSize:
-      name: page[size]
+    businessCenter:
       in: query
-      required: false
-      description: Number of results to return
+      name: filter[businessCenter]
+      description: Search by 4-character business center name
+      required: true
       schema:
-        type: integer
-        minimum: 1
-        maximum: 500
-        default: 25
+        type: string
+        example: ABCD
+        pattern: ^.{4}$
```

Comment:
I feel this should be stricter than just `.`. You can do `[a-zA-Z0-9]` for any alphanumerical character.

Reply:
Good idea.